src/olympia/addons/decorators.py

```diff
@@ -21,7 +21,7 @@ def owner_or_unlisted_reviewer(request, addon):
 
 def addon_view(f, qs=Addon.objects.all):
     @functools.wraps(f)
-    def wrapper(request, addon_id=None, app_slug=None, *args, **kw):
+    def wrapper(request, addon_id=None, *args, **kw):
         """Provides an addon given either an addon id or an addon slug."""
```

Comment:
Here the `app_slug` is removed but the docstring still mentions it, so I'm doubly confused.